sys/mips/conf/CHERI_PURECAP_DE4_MFS_ROOT_FUZZ

```diff
@@ -0,0 +1,19 @@
+#
+# CHERI -- Benchmark coverage configuration for pure capability kernel for
+# the SRI/Cambridge "CHERI" (Capability Hardware Enhanced
+# RISC Instructions) FPGA soft core, as configured in its Terasic DE-4
+# reference configuration - 128bit version.
+#
+# $FreeBSD$
+#
+
+include		"CHERI_PURECAP_DE4_MFS_ROOT"
+ident		CHERI128_PURECAP_DE4_MFS_ROOT_FUZZ
+
+# Disable slow things
+include "std.BENCHMARK"
```

Comment:
Do you also want a fuzzing kernel that has assertions enabled?

Reply:
The benchmark here was used mostly to have decent run times with kcov, in theory the fuzz kernel should probably only exist with assertions enabled

Reply:
This was actually duplicating another variant. Removed it.